GithubApp/Controllers/FilterViewController.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  FilterViewController.swift
+//  GithubApp
+//
+//  Created by BrainX on 09/06/2022.
+//
+
+import UIKit
+
+class FilterViewController: UIViewController {
+
+    @IBOutlet var filterView: FilterView!
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        filterView.filterTableView.delegate = self
+        filterView.filterTableView.dataSource = self
+    }
+    let languagesData = ["Python","Interior Painting","Electrician","Inappropriate client behavior","Gardener","Carpenter","House Cleaning"]
```

Comment:
Pragma marks missing. Each property, method and extension should be under some pragma mark.